server/routes/user.js

```diff
@@ -35,4 +35,24 @@ router.post('/user/logout', authenticate, (req,res) => {
     })
 })
 
+router.delete('/admin/delete/:uuid', authenticate, (req,res) => {
+    userLibs.deleteUser(req).then((success) => {
+        res.status(success.status).send(success);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    })
+})
+
+router.delete('/user/delete/:uuid', authenticate, (req,res) => {
```

Comment:
this will be deactivate not delete and word should be deactivate not deactive this applys every where

Reply:
Done